diff --git a/setup.py b/setup.py
index 45491bc3..68c8dad6 100755
--- a/setup.py
+++ b/setup.py
@@ -56,7 +56,7 @@ def is_new_osx():
 
 
 # -fopenmp ?
-COMPILE_OPTIONS = ['-O2', '-Wno-strict-prototypes', '-Wno-unused-function']
+COMPILE_OPTIONS = ['-O2']
 # -fopenmp ?
 LINK_OPTIONS = []
 
@@ -134,7 +134,7 @@ for ext_module in ext_modules:
     ext_module.extra_link_args = LINK_OPTIONS
 
 
-with open('README.md', 'r') as fh:
+with open('README.md', 'r', encoding='utf8') as fh:
     long_description = fh.read()
 
 
